lib/erlef/inputs.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Erlef.Inputs do
+  @moduledoc """
+  Erlef.Inputs provides re-useable input validations
+  """
+
+  # TODO: Expand this to check the entire spec
```

Comment:
It is absolutely and I have done the regex route before 😬. I very much want to avoid regex for user input for security reasons, but also felt like at least a simple check would be good. Whether I actually come back and expand this is another story 😁